Doc/library/uuid.rst

```diff
@@ -10,10 +10,11 @@
 
 --------------
 
-This module provides immutable :class:`UUID` objects (the :class:`UUID` class)
-and the functions :func:`uuid1`, :func:`uuid3`, :func:`uuid4`, :func:`uuid5`,
-:func:`uuid6`, and :func:`uuid8` for generating version 1, 3, 4, 5, 6,
-and 8 UUIDs as specified in :rfc:`9562` (which supersedes :rfc:`4122`).
+This module provides immutable :class:`UUID` objects (the :class:`UUID` class) and
+the :ref:`functions <uuid-constructor-functions>` for generating UUIDs corresponding
```

Comment:
```suggestion
This module provides immutable :class:`UUID` objects (the :class:`UUID` class)
and :ref:`functions <uuid-functions>` for generating UUIDs corresponding
```

«constructor» has a different meaning related to class, these are rather factory functions: they give a simplified interface to instantiate a class. But I suggest that just saying functions is good enough.

Reply:
I will name them `uuid-factory-functions`. (The `uuid` prefix is for the module, not for the fact that we're creating UUID objects; it's to disambiguate Sphinx labels)